buildout.cfg

```diff
@@ -66,17 +66,26 @@ zc.buildout = 2.4.5
 zc.recipe.cmmi = 1.3.6
 zc.recipe.egg = 2.0.3
 zest.releaser = 5.7
+numpy = 1.10.1
```

Comment:
numpy en pandas versies zijn de versies die uit jouw syseggs komen. Op een andere compu kan het net anders zijn. Beter om `numpy =` te doen zodat elke sysegg versie OK is.

Ik kan het wel een keertje uitleggen.